ios/RCTTWSerializable.m

```diff
@@ -8,18 +8,40 @@
 
 #import "RCTTWSerializable.h"
 
-@implementation TVIParticipant(RCTTWSerializable)
+@implementation TVIRemoteParticipant(RCTTWSerializable)
 
 - (id)toJSON {
-  return @{ @"identity": self.identity };
+    return @{ @"identity": self.identity };
```

Comment:
Please, review indentation in changed files. The project is using 2 spaces for indentation.